lib/index.js

```diff
@@ -352,10 +366,10 @@ internals.getExistsValues = function (type, exists) {
             return (value.isContext ? '$' : '') + value.key;
         }
 
-        return value;
+        return JSON.stringify(value);
```

Comment:
Did this here as there's no way to differentiate between the key values returned above and actual strings in the template.